src/briefcase/integrations/android_sdk.py

```diff
@@ -1366,27 +1370,37 @@ def start_emulator(
                     while not adb.has_booted():
                         if emulator_popen.poll() is not None:
                             raise BriefcaseCommandError(
-                                error_msg.format(
+                                failed_startup_error_msg.format(
                                     prologue="Android emulator was unable to boot!"
                                 )
                             )
 
                         # Try again in 2 seconds...
                         self.sleep(2)
         except BaseException as e:
-            # if the emulator exited, this should return its output immediately;
-            # if it is still running, this will quickly time out and print nothing.
-            with suppress(subprocess.TimeoutExpired):
+            self.tools.logger.info("Emulator output log", prefix=self.name)
```

Comment:
I'd be inclined to bump this to warning or error. As currently rendered, it looks a bit like "normal output"; but if we're at this point, we're already in "something is in the process of breaking" mode.

It might also be worth highlighting that it's not just the output log - it's "output because of error".

Reply:
Yeah; I think I'm hesitant to make it red error output because that will automatically become the first thing the user looks at and perceived as the most important....and it might be. Mostly, I wanted to direct the user to where I wanted them to start reading...at least in the case of CTRL+C. For the other failure case, it's just a wall of red text.

Anyway, I think changing the emulator output heading to a yellow warning and saying `Emulator Output Log for Startup Failure` instead.